mods/seasonal/moves.js

```diff
@@ -83,6 +83,32 @@ let BattleMovedex = {
 		target: "normal",
 		type: "Normal",
 	},
+	// 5gen
+	toomuchsaws: {
+		accuracy: 100,
+		basePower: 85,
+		basePowerCallback: function (pokemon, target, move) {
+			if (target.newlySwitched) {
+				return move.basePower * 2;
+			}
+			return move.basePower;
```

Comment:
Im pretty sure this is supposed to be like pursuit, not "if they switched last turn"

Reply:
```Effects: Power doubles if foe switches out```
i went by that, ask them if they meant something else then please

Reply:
ok your correct